xpdacq/beamtime.py

```diff
@@ -365,17 +365,8 @@ def __init__(self,sample, scanplan):
         self.type = 'sc'
         self.sp = scanplan
         self.sa = sample
-        self.md = self.sp.md
+        self.md = dict(self.sp.md) # create a new dict copy.
```

Comment:
I got this idea from Tom's `collect_time_series` function [here](https://github.com/xpdAcq/xpdAcq/blob/master/xpdacq/xpdacq.py#L418-L419) and this logic pass unittest cases of 1) directly playing md in `Scan` class and 2) passing `Scan` to `prun` and `dark`.

I can change code to the way describe above so that it is more intuitive while reading.